thrive-refugee/thrive-refugee
# Add a few sample file uploads to the fixtures

Fixes #45 

To test:
- verify the 1st sample family has zero files
- verify the 2nd sample family has two files
- verify the 3rd sample family has one file
